openmetadata-ui/src/main/resources/ui/cypress/e2e/Pages/DataInsightSettings.spec.ts

```diff
@@ -124,9 +124,10 @@ describe(
 
       // Adding a manual wait to allow some time between deploying the pipeline and triggering it
       // eslint-disable-next-line cypress/no-unnecessary-waiting
-      cy.wait(1000);
+      cy.wait(2000);
```

Comment:
@ShaileshParmar11 can we think something else here instead of Manuel wait?

Reply:
After clicking the deploy button, there's a background process to deploy into Airflow, which isn't under our control. The backend handles this process, and the UI doesn't have a way to determine if it's deployed. That's why I'm relying on a manual waiting approach. If you have any ideas, please feel free to share them.